diagnostics/model_test/mnist_cnn.json

```diff
@@ -0,0 +1,10 @@
+{
+    "model_name": "mnist_cnn",
+    "frozen_graph_location": "/nfs/site/home/mingshan/frozen_graphs/frozen_mnist.pb",
+    "input_tensor_name": "import/Placeholder:0",
+    "output_tensor_name":"import/fc2/yconv:0",
+    "l1_norm_threshold": 0.01,
+    "l2_norm_threshold": 0.01,
+    "inf_norm_threshold": 0.01,
+    "input_dimension" : 784
+}
```

Comment:
The graph can have multiple inputs, and each input can have multiple dimensions.
Ideally we want to support something like this:
[[28,28], [5,5], [3,3,10]]
to indicate 3 input dimensions